etc/spack/defaults/windows/bootstrap-resource.yaml

```diff
@@ -0,0 +1,16 @@
+bootstrap-resource:
```

Comment:
As I compare this and `bootstrap.yaml`, I think it would actually be worthwhile to merge this into the `bootstrap` schema as a subsection - does that seem agreeable?

Reply:
If that won't make it too busy I don't see why not.